### ISSUE TITLE
how to upgrade 

### ISSUE THREAD

**Issue by Hefziben** [hyperledger/composer#1967]
Hello Team, sorry for my lakc of knowledge, but could you please tell me how to do an upgrade of composer?

**Comment by davidkel** [comment]
Issue #1842 exists as we only have a small amount of detail about updating at present. Can you provide more detail about your environment and what you mean by upgrade ?

**Comment by Hefziben** [comment]
Context: 
I am testing a a business network I created, I run the Composer-rest-server and all worked fine, then I proceeded use the yo hyperledger composer to create the skeleton of the angular app, however, now the angular app is showing in the local browser, however, the composer-rest- server is not.

Expected Behavior: 
I should start the composer-rest- server 

Actual Behavior:

I get this message; 
scovering types from business network definition ...
Connection fails: Error: Error trying to ping. Error: Error trying to query chaincode. Error: Connect Failed
It will be retried for the next request.
Exception: Error: Error trying to ping. Error: Error trying to query chaincode. Error: Connect Failed
Error: Error trying to ping. Error: Error trying to query chaincode. Error: Connect Failed
    at _checkRuntimeVersions.then.catch (/home/node/.nvm/versions/node/v6.11.2/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:696:34)

Your Environment
composer-cli@0.11.3
generator-hyperledger-composer@0.11.3
composer-rest-server@0.11.3
Docker version 17.06.0-ce, build 02c1d87
docker-compose version 1.13.0, build 1719ceb
